docs/src/pages/[platform]/connected-components/liveness/customization/customization.system-clock-offset.react.mdx

```diff
@@ -14,25 +14,30 @@ function MyComponent() {
   const [createLivenessApiData, setCreateLivenessApiData] = React.useState(null);
   
   /* 
-   * 1. Check whether the difference between server time and device time
-   * is greater than 5 minutes, and if so, return the offset in milliseconds
-   * This logic should be adjusted based on the server response and your use case
+   * 1. Check whether the difference between server time and device time is
+   *  greater than or equal to 5 minutes, and if so, return the offset in milliseconds.
+   *  This logic should be adjusted based on the server response and use case
   */ 
-  const getSystemClockOffset = (serverDate: string | undefined) => {
+  const getSystemClockOffset = (serverTime: number) => {
+    const maxSupportedClockSkew = 5 * 60 * 1000; // 5 minutes
     const deviceTime = Date.now();
-    const serverTime = new Date(serverDate).getTime();
     const delta = serverTime ? serverTime - deviceTime : 0;
-    return Math.abs(delta) > maxSupportedClockSkew ? delta : undefined;
+    return Math.abs(delta) >= maxSupportedClockSkew ? delta : undefined;
   }
   
   React.useEffect(() => {
     /*
-    * This should be replaced with your own API call to create a session
+    * Replace with your own API call to create a session
     */
     const response = await fetch(`/api/createSession`);
     const body = await response.json(); // { sessionId: 'mockSessionId' } 
-    const systemClockOffset = getSystemClockOffset(response.headers['date'])
-    setCreateLivenessApiData(body, systemClockOffset);
+    /*
+    * Replace serverDate with the actual server date,
+    * which can be retrieved from the response headers or your customer backend.
+    */
+    const serverTime = Date.now();
```

Comment:
We should probably show default example of `const serverTime = response.headers['date']`, right?

Reply:
Yeah mstm